src/Host/Client/Impl/Microsoft.R.Host.Client.csproj

```diff
@@ -13,7 +13,7 @@
   <ItemGroup>
     <PackageReference Include="MicroBuild.Core" Version="0.2.0" />
     <PackageReference Include="Microsoft.AspNetCore.WebSockets.Protocol" Version="0.1.0" />
-    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="4.3.0" />
+    <PackageReference Include="System.Net.Http.WinHttpHandler" Version="4.4.0" />
```

Comment:
I had type resolution problems because of this change. Do you need it?

Reply:
Not really. Seems to work, but I can roll back if you had issues.